src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,13 +1,11 @@
 package mate.academy.spring.dto.request;
 
 import javax.validation.constraints.Min;
-import javax.validation.constraints.NotNull;
 import javax.validation.constraints.Size;
 
 public class CinemaHallRequestDto {
-    @Min (value = 20)
+    @Min(value = 20)
```

Comment:
I meant adding a not-null constraint, and min capacity is actually either not needed or not so big